app/javascript/components/components/TopCrawls.js

```diff
@@ -22,7 +23,7 @@ class TopCrawls extends Component {
           </NavLink>
       </Card>
       <Card className= "top-crawls-card">
-      <NavLink to= 'popularcrawls/7'>
+      <NavLink to= 'popularcrawls/6'>
```

Comment:
this looks hardcoded. Will it always be expected to be `popularcrawls/6`?

Reply:
Yes for this one it will! Because it’s just showing our fake “top crawls” on the home page for now